yarn.lock

```diff
@@ -1357,6 +1357,11 @@
     "@types/prop-types" "*"
     csstype "^3.0.2"
 
+"@types/resize-observer-browser@^0.1.5":
```

Comment:
i don't think this should be here. make sure you checkout this update and merge in the latest master.